### DISCUSSION TITLE
Add new multiline card form?

### DISCUSSION THREAD

**Discussion by Lab360-inc** [flutter-stripe/flutter_stripe#325] [Q&A] [answered]
In the release it says Added new multiline card form. Anyone got any details on it? does it apply to cardfield or paymentsheet?

**Comment by remonh87** [comment, marked as answer]
It applies to the CardField see: https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/card_payments/no_webhook_payment_cardform_screen.dart
